projects/storefrontapp-e2e-cypress/cypress/plugins/index.js

```diff
@@ -3,16 +3,6 @@ const fs = require('fs');
 
 module.exports = (on, config) => {
   on('file:preprocessor', cypressTypeScriptPreprocessor);
-
-  /* Set exact timestamp to be shared in all spec files */
-  config.env.TIMESTAMP = Date.now() - 1535535333333;
-  return config;
-};
-
-/**
- * Read file but return null if no file found (instead of failing the test).
- */
-module.exports = (on, config) => {
```

Comment:
this is what broke `cy.requireLoggedIn()` <-- without an input param, where the user's timestamp is always `_undefined`, therefore might have been breaking overall e2es

Reply:
You can verify the facts by going to `projects/storefrontapp-e2e-cypress/cypress/support/require-logged-in.commands.ts` and append `console.log('username', username);` after line 90.

```
    const username =
      account.registrationData.email ||
      generateMail(account.user, options.freshUserOnTestRefresh);

    console.log('username', username);

```

You'll see the logged in user in develop will be `cypress_user_somethingsomething_undefined@sapcx.com` instead of a proper timestamp. `_undefined`